src/countries/at.js

```diff
@@ -25,11 +25,10 @@ var fixLongAtPhone = tools.fixPhoneBuilder(11, 14, longConfig);
 
 var fixAtPhone = function(phone) {
   phone = tools.getSanitizedPhone(phone)
+  phone = phone.replace(/\+*/, '')
```

Comment:
\+* means zero or more occurences of +
do you really want to replace zero plus symbols? :)
I think you should replace * to +